packages/contracts-bedrock/src/cannon/interfaces/IPreimageOracle.sol

```diff
@@ -1,41 +1,89 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.15;
+pragma solidity ^0.8.0;
```

Comment:
We should update `check-interfaces` to make sure the pragma is correct

Reply:
Going to do this as part of a separate PR

Reply:
PR #11740